framework/src/security/security_utils.c

```diff
@@ -266,8 +267,11 @@ int secutils_convert_path_s2h(const char *path, uint32_t *slot)
 	if (!strncmp(path, SS_PATH, sizeof(SS_PATH) - 1)) {
 		*slot = atoi(&path[3]);
 		return 0;
+	} else if (!strncmp(path, FACTORY_PATH, sizeof(FACTORY_PATH) - 1)) {
+		*slot = FACTORYKEY_ARTIK_CERT;
```

Comment:
This FACTORYKEY_ARTIK_CERT definition belongs to "ARTIK", which is not appropriate in this framework, in my opinion. It would be better to use this FACTORYKEY_ARTIK_CERT in wrapper layer.

Reply:
In future, we should decide how to pass cert name or key index to hal.